spec/views/support_users/publisher_ats_api_clients/index.html.slim_spec.rb

```diff
@@ -13,9 +16,9 @@
     expect(rendered.html.css("td").map(&:text))
       .to eq(
         ["Big MAT ATS",
-         api_client.created_at.to_fs,
-         "0",
+         vacancy_one.created_at.to_fs,
          "2",
+         "1",
```

Comment:
[Minor] Not too sure what these strings are, the it block says 'it "shows some data"' which doesn't help with context either!

Reply:
There is room for improvement in the wording/context.

Going to release it as it is since the feature is broken at the moment, so better get it out ASAP.